.githooks/pre-commit

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+exec < /dev/tty
```

Comment:
Guessing that we'll add the 'npm run lint' command later?

Reply:
Yes that'll be when we add the node js code with the lint library.